### PR TITLE
feat(Video Stitcher):Add Media CDN samples. Move Akamai samples to th…

### PR DESCRIPTION
**Pull request by irataxy**
…eir own files.

Fixes b:248290230

### CODE REVIEW

File: media/stitcher/api/Stitcher.Samples.Tests/UpdateCdnKeyTests.cs [hunk 1]
```diff
@@ -33,17 +33,9 @@ public UpdateCdnKeyTest(StitcherFixture fixture)
...
         public void UpdatesCdnKey_CloudCdn()
         {
-            var result = _updateSample.UpdateCdnKey(_fixture.ProjectId, _fixture.LocationId, _fixture.TestCloudCdnKeyId, _fixture.UpdateHostname, _fixture.CloudCdnKeyName, _fixture.UpdatedCloudCdnTokenKey, null);
+            var result = _updateSample.UpdateCdnKey(_fixture.ProjectId, _fixture.LocationId, _fixture.TestCloudCdnKeyId, _fixture.UpdatedHostname, _fixture.KeyName, _fixture.CloudCdnPrivateKey, false);
             Assert.Equal(_fixture.TestCloudCdnKeyId, result.CdnKeyName.CdnKeyId);
-            Assert.Equal(_fixture.UpdateHostname, result.Hostname);
-        }
-
-        [Fact]
-        public void UpdatesCdnKey_Akamai()
```
Comment (amanda-tarafa) [inline media/stitcher/api/Stitcher.Samples.Tests/UpdateCdnKeyTests.cs:42]: It's the media one not testable?

**Review by amanda-tarafa — COMMENTED** [review]
One question only, otherwise LGTM

File: media/stitcher/api/Stitcher.Samples.Tests/UpdateCdnKeyTests.cs (hunk 1 above)
Comment (irataxy) [reply media/stitcher/api/Stitcher.Samples.Tests/UpdateCdnKeyTests.cs:42]: Good catch. I added the test for Media CDN.

**Review by amanda-tarafa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**